feat(curriculum): restructured expense tracker python project (#52639)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-an-expense-tracker/658252f6b1526d57b103d48a.md

---
id: 658252f6b1526d57b103d48a
title: Step 30
challengeType: 20
dashedName: step-30
---

# --description--

> Inside the `if` statement, you should ask the user to enter the amount for the expense and store it in a variable.

Reviewer: ```suggestion
Inside the `if` statement, prompt the user to enter the amount for the expense and store it in a variable.
```
